backend/common/kernel.py

```diff
@@ -22,126 +27,98 @@
 from backend.common.utils import csv_to_json
 from backend.common.ai_drive import dl_tabular_drive, ml_drive, dl_img_drive
 from backend.dl.detection import detection_img_drive
-from backend.aws_helpers.dynamo_db_utils.execution_db import updateStatus
 
 
 def router(msg):
     """
     Routes the message to the appropriate training function.
     """
     print("Message received")
```

Comment:
might be good to use the `logger` module to log behavior. In our case, 

```
logger.info("Message received")
```

Docs: [here](https://realpython.com/python-logging/)